src/menutemplate.js

```diff
@@ -17,7 +17,13 @@ var MenuTemplate = function () {
       submenu: [
       {
         label: 'About Kitematic',
-        selector: 'orderFrontStandardAboutPanel:'
```

Comment:
Sorry, I meant it _is_ necessary for OS X. The switch would be to check with arch you're on. ;)